### PR TITLE
add import_blocklist command

### LINKED ISSUE #12460: Import current blocklist data
Part of this [PRD](https://docs.google.com/document/d/14_Xz_r46yQF4wN7BtrgWfWBdjEqCVdY7d8T_ZjVf1OE/edit). [Mocks](https://docs.google.com/drawings/d/1K6mpb_06oB1MYpQF3gagolWoDCMEHK8Sc_LMvMH6ass/edit).

This will require coordination with the admin reviewers, because many blocklist entries will need to be broken down into a format compatible with what will be stored on AMO. Please ping me in advance of this being needed.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #12460

### CODE REVIEW

File: src/olympia/blocklist/tests/blocklists/blocklist.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
...ee\\.myformsfinder\\.com)|(_dqMembersAlt_@www\\.downspeedtest\\.com)|(_rcMembers_@www\\.extsb\\.searchbetter\\.com)|(_piMembers_@install\\.mysporttab\\.com)|(_phMembers_@install\\.streamfrenzy\\.com)|(_pfMembers_@install\\.myvideotab\\.com)|(_qxMembers_@install\\.utili-site\\.com)|(_qyMembers_@install\\.cryptoconvertertab\\.com)|(_r4Members_@install\\.salah-time\\.com)|(_r9Members_@install\\.speed-exam\\.com)|(_rbMembers_@Install\\.movie-canal\\.com)|(_poMembers_@www\\.linkuryds1\\.com)|(_l9Members_@www\\.plaskodss3\\.com)|(_ppMembers_@www\\.firstofferzds1\\.com)|(_qkMembers_@www\\.exploreads\\.com)|(_kfMembers_@www\\.bluecpads\\.com)|(_k2Members_@www\\.arcadejetds\\.com)|(_qiMembers_@tvguru-lp\\.olympuswaymarketing\\.com)|(_p8Members_@www\\.easywatch\\.online)|(_kcMembers_@www\\.utilitytab\\.com)|(_kdMembers_@www\\.utilitytab\\.com)|(_qrMembers_@www\\.install\\.utilitytab\\.com)|(_qsMembers_@www\\.utilitytab\\.com)|(_izMembers_@www\\.geteasyarcade\\.com)|(_qgMembers_@www\\.getinstantpdf\\.com)|(_p7Members_@www\\.tigersdeal\\.com)|(_jgMembers_@www\\.greathippo\\.com)|(_jdMembers_@www\\.greathippo\\.com)|(_juMembers_@arcadejet\\.com)|(\\{90ac1d06-caf8-46b9-9325-59c82190b687\\}))$/",
```
Comment (diox) [inline src/olympia/blocklist/tests/blocklists/blocklist.json]: Not sure how I feel about having actual blocklist content in this test file. I see the appeal, and it's public anyway, but I am not sure we want to republish that in our repos

Comment (eviljeff) [reply src/olympia/blocklist/tests/blocklists/blocklist.json]: I could anonymize some of the guids I guess?  I was just trying to keep the test data authentic.

Comment (diox) [reply src/olympia/blocklist/tests/blocklists/blocklist.json]: Yeah, maybe anonymize it. I kinda don't like publishing those domain names in a file in our repos if we can avoid it.

Comment (eviljeff) [reply src/olympia/blocklist/tests/blocklists/blocklist.json]: went through and slightly changed each guid so they won't match the real domain names we blocked https://github.com/mozilla/addons-server/pull/13043/commits/1d9b451b70e2e0c2962bcbad99f40ff17bf1bf7e

File: src/olympia/blocklist/management/commands/import_blocklist.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+from olympia.blocklist.models import Block
+from olympia.blocklist.utils import block_activity_log_save
+from olympia.users.utils import get_task_user
+
+
+log = olympia.core.logger.getLogger('z.amo.blocklist')
+
+
+class Command(BaseCommand):
+    help = ('Populate AMO blocklist by import v2 JSON blocklist from kinto.')
```
[the remark this answers is not in the document]
Comment (eviljeff) [reply src/olympia/blocklist/management/commands/import_blocklist.py]: https://github.com/mozilla/addons-server/pull/13043/commits/49a153c163aca74f7bab2264678e97935d4af807

**Pull request merged**